backend/app/api/endpoints/api_keys.py

```
@@ -0,0 +1,119 @@
+    APIKeyCreate,
+    APIKeyCreatedResponse,
+    APIKeyListResponse,
+    APIKeyResponse,
+)
+
+router = APIRouter()
+
+
+@router.get("", response_model=APIKeyListResponse)
+async def list_api_keys(
+    db: Session = Depends(get_db),
```

Ruff (0.14.8)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)